test_roles/confluent.test/tasks/check_connector.yml

```diff
@@ -3,7 +3,7 @@
   confluent.platform.kafka_connectors:
     connect_url: "{{ connect_url_input }}"
     active_connectors: "{{ active_connectors_input }}"
-    token: "{{ authorization_token }}"
+    token: "{% if rbac_enabled or kafka_connect_oauth_enabled %}{{ authorization_token }}{% else %}{{none}}{% endif %}"
```

Comment:
what was the need to add this? won't this token just be empty when it's non rbac and non oauth?

Reply:
It cause problems when sending empty variable. Instead we use none. Even in the deploy_connectors file we use similar logic to send none in case of unauthenticated connectors